core/src/main/java/com/linecorp/armeria/server/JacksonModuleProvider.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
```

Comment:
Any chance we're gonna use it for clients?

Reply:
I also think about it. It is possible if `WebClient` provides automatic conversion for JSON.
It would be better to move to `common` cause we gonna provide it before long. 😆